decidim/decidim
# Fix unpublished content displayed in search engine

#### :tophat: What? Why?
Non-visible resources are displayed in the search.
Each resource has a parent, which is a component, which has a parent: a participatory space.
When the component or participatory space is unpublished, the resource continues to be published. 
This is due to the fact that indexing in the search is only done when the resource is updated or created. When it concerns its parents, these functions are not called.
In this context, this feature adds a feature that invites resources to check if they are visible or not. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #6491

#### Testing
* Create proposals
* Unpublished participatory space
* Check that proposal is unpublished

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.


:hearts: Thank you!
